features/environment_secret.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: UTF-8 -*-
 import os
 
-HIPCHAT_LOGIN = os.environ.get("HCLOGIN", "tim+test@bortnik.org")
-HIPCHAT_PASS = os.environ.get("HCPASS", "ntcn987654321")
+HIPCHAT_LOGIN = os.environ.get("HCLOGIN", "ighost1996@gmail.com")
+HIPCHAT_PASS = os.environ.get("HCPASS", "qwert12345")
```

Comment:
Please use environment variables, any password that was pushed to open repo should be changed ASAP